website/source/docs/auth/aws.html.md

```diff
@@ -645,6 +645,11 @@ $ vault login -method=aws header_value=vault.example.com role=dev-role-iam \
         aws_security_token=<security_token>
 ```
 
+The region used defaults to `us-east-1`, but you can specify a custom region like so:
+```
+$ vault login -method=aws region=us-west-2 role=dev-role-iam
```

Comment:
I would suggest adding a comment here that the region used needs to correspond to the region of the sts_endpoint configured on the Vault role.

Reply:
Added in https://github.com/hashicorp/vault/pull/6268/commits/10551cb1d251e249b22b74d1aad848bbbce675ab